test/script-test.js

```diff
@@ -345,4 +346,45 @@ describe('Script', function() {
       });
     }
   }
+
+  for (const isSorted of [true, false]) {
+    for (const isWitness of [true, false]) {
+      for (let n = 1; n <= 22; n++) {
```

Comment:
```suggestion
      for (let n = 1; n <= 21; n++) {
```
I think 21 would suffice